.circleci/config.yml

```diff
@@ -870,28 +872,21 @@ jobs:
           command: |
             echo "Building all contracts under ./contracts"
             docker run --volumes-from with_code cosmwasm/rust-optimizer:0.10.8 ./contracts/*/
-            docker cp with_code:/code/artifacts .
-      - run:
-          name: Install check_contract
-          # Uses --debug for compilation speed
-          command: cargo install --debug --features iterator --path ./packages/vm --example check_contract -- cosmwasm-vm
       - run:
           name: Check development contracts
           command: |
-            for W in ./artifacts/*.wasm
-            do
-              echo -n "Checking `basename $W`... "
-              check_contract $W
-            done
-      - run:
-          name: Publish artifacts on GitHub
-          command: |
-            TAG="$CIRCLE_TAG"
-            TITLE="$TAG"
-            BODY="Attached there are some build artifacts generated at this tag. Those are for development purposes only! Please use crates.io to find the packages of this release."
-            ghr -t "$GITHUB_TOKEN" \
-              -u "$CIRCLE_PROJECT_USERNAME" -r "$CIRCLE_PROJECT_REPONAME" \
-              -c "$CIRCLE_SHA1" \
-              -n "$TITLE" -b "$BODY" \
-              -delete \
-              "$TAG" ./artifacts/
+            echo "Checking all contracts under ./artifacts"
+            docker run --volumes-from with_code rust:1.50.0 /bin/bash -e -c 'cargo install --debug --features iterator --path ./code/packages/vm --example check_contract -- cosmwasm-vm; for W in ./code/artifacts/*.wasm; do echo -n "Checking `basename $W`... "; check_contract $W; done'
```

Comment:
Will put all this (`cargo install --debug --features iterator --path ./code/packages/vm --example check_contract -- cosmwasm-vm; for W in ./code/artifacts/*.wasm; do echo -n "Checking `basename $W`... "; check_contract $W; done'`)
into a `check_contracts.sh` helper script, for "ergonomics".